game/scripts/npc/items/item_silver_edge.txt

```diff
@@ -56,17 +56,17 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_damage"                                    "30 40 50 60 70"
+        "bonus_damage"                                    "30 40 50 65 85"
```

Comment:
silver edge has the worst scaling stats of any item in the game, if you think its too far i'm more than happy to change it back, but these changes were mostly insignificant until level 4/5 because right now I don't think there's much incentive to upgrade silver edge; it really seems like it has the most value at level 1 as a backpack item. sure break is powerful, but into a satanic its a 3 second break. not sure silver edge can really be classified as invis +break

just say the word and i'll change it back, but it could be an idea to scale the break 4/4/5/5/6 (4/4.5/5/5.5/6 ? no clue)

Reply:
I think you should revert the buff to bonus damage and bonus attack speed.